src/normalinversechisq.jl

```diff
@@ -0,0 +1,62 @@
+# Based on Murphy (2007) Conjugate Bayesian Analysis of the Gaussian
+# distribution.  Equivalent to the Normal-Inverse Gamma under the follow
+# re-parametrization:
+#
+# m0 = μ
+# v0 = 1/κ
+# shape = ν/2
+# scale = νσ2/2
+#
+# The parameters have a natural interpretation when used as a prior for a Normal
+# distribution with unknown mean and variance: μ and σ2 are the expected mean
+# and variance, while κ and ν are the respective degrees of confidence
+# (expressed in "pseudocounts").
+
```

Comment:
As docstring?

Reply:
good idea. I guess I didn't because none of the other ones have them.  It's probably worth overhauling all the docs and using docstrings